substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2270,8 +2270,14 @@ impl pallet_alliance::Config for Runtime {
 	type RetirementPeriod = RetirementPeriod;
 }
 
+frame_support::parameter_types! {
+	// Change here if you modified the Genesis config or added a pallet etc:
+	pub StorageRoot: Option<Vec<u8>> = array_bytes::hex2bytes("4ca13c91ff2efada4eade93a18070c593df9ee603370082eb4d837de6853d168").unwrap().into();
```

Comment:
Not sure how often this will break, but it is the safest way to ensure that the Benchmarks always run with the same storage root.